credentials/alts/alts.go

```diff
@@ -307,7 +307,7 @@ func compareRPCVersions(v1, v2 *altspb.RpcProtocolVersions_Version) int {
 // agreed on.
 func checkRPCVersions(local, peer *altspb.RpcProtocolVersions) (bool, *altspb.RpcProtocolVersions_Version) {
 	if local == nil || peer == nil {
-		grpclog.ALTS.Error("invalid checkRPCVersions argument, either local or peer is nil.")
```

Comment:
Same for ALTS

Reply:
This has been replaced with an in-package defined "logger" var.